lib/Auth/Process/privacyidea.php

```diff
@@ -87,11 +87,21 @@ public function process(&$state)
     }
 
     private function maybeTryFirstAuthentication($state) {
-        return $this->serverconfig['tryFirstAuthentication']
-            && sspmod_privacyidea_Auth_utils::authenticate(
+        if (!$this->serverconfig['tryFirstAuthentication']) {
+            return false;
+        }
+
+        try {
```

Comment:
Please add

```php
$state['privacyidea:privacyidea:authenticationMethod'] = "authprocess";
```

Reply:
I am not 100% familiar with simplesamlphp and this module, but maybe we can set `$state['privacyidea:privacyidea:authenticationMethod'] = "authprocess";` in auth proc filter at the point where we surely know that it's enabled and configured? For example, after we check for passive authentication. In this case we don't have to set it in `maybeTryFirstAuthentication` and `openOtpform` methods.

Reply:
You are right, lets put it there in the process method and remove the statement in `openOtpform` and in `maybeTryFirstAuthentication`. Could you do that? Thank you!

Reply:
My colleague merged too early. I did the change in https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/117